change/@internal-react-components-3f6426b0-9fb2-4856-aff3-40edc7924dfb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Going by changes this should be a patch based on the reordering of participants in the read receipts